pkgs/development/php-packages/smbclient/default.nix

```diff
@@ -0,0 +1,13 @@
+{ buildPecl, lib, pkgs }:
+buildPecl {
+  pname = "smbclient";
+  version = "1.0.4";
+  sha256 = "07p72m5kbdyp3r1mfxhiayzdvymhc8afwcxa9s86m96sxbmlbbp8";
+
+  LIBSMBCLIENT_INCDIR = "${pkgs.samba.dev}/include/samba-4.0";
```

Comment:
After reading [config.m4](https://github.com/eduardok/libsmbclient-php/blob/1.0.4/config.m4) I had the feeling an upstream fix might be required. I don't have the skills for this type of fix, so I just used this hack instead.

@jtojnar would know what to do...

Reply:
The package seems to use some ancient macros. Ideally it would switch to pkg-config.

And it seems our @bachp has been also looking into this: https://github.com/eduardok/libsmbclient-php/pull/66

Reply:
@jtojnar a bit hacky, sure, but do you see any problem proceeding with this for now until upstream is able to respond? I think I'm going to need this soon and would like to merge + possibly backport this extension.

Reply:
Should be fine. Maybe add a comment linking to the upstream PR though.

Reply:
Great. Thanks for your review @jtojnar.